src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
+scan_proper_plugins.load_setuptools_entrypoints('scancode_plugins')
```

Comment:
Why is this needed here now? the current plugins are only about the output formatting at this stage.